content/blog/dplyr-1-0-0/index.Rmd

```diff
@@ -0,0 +1,116 @@
+---
+title: dplyr 1.0.0 available now!
+output: hugodown::hugo_document
+
+description: > 
+  dplyr 1.0.0 is now available from CRAN!
+author: Hadley Wickham
+date: '2020-06-01'
+slug: dplyr-1-0-0
+
+photo:
+  url: https://unsplash.com/photos/W8BNwvOvW4M
+  author: Helinton Fantin
+
+categories:
+  - package
+tags:
+  - dplyr
+---
+
+I'm very excited to announce the ninth and final blog post in the [dplyr 1.0.0 series](/tags/dplyr): [dplyr 1.0.0](http://dplyr.tidyverse.org/) is now available from CRAN! Install it by running:
+
+```{r, eval = FALSE}
+install.packages("dplyr")
+```
+
+## New features
+
+dplyr 1.0.0 is chock-a-block with new features; so many, in fact, that we can't fit them all into one post. So if you want to learn more about what's new, we recommend reading our existing series of posts:
+
+-   [Major lifecycle changes](/blog/2020/03/dplyr-1-0-0-is-coming-soon/). 
+    This post focusses on the idea of the "function lifecycle" which helps 
+    you understand where functions in dplyr are going. Particularly important
+    is the idea a "superseded" function; this function is not going away, 
```

Comment:
```suggestion
    is the idea a of "superseded" function. A superseded function is not going away, 
```

Reply:
My language sounds clunkier, but the post-semicolon _this function_ almost makes it sound like there is a function `superseded()`.

Either way, the "of" should definitely go in.

Reply:
But presumably _before_ the "a" 😆